docs/source/demos/fraud.ipynb

```diff
@@ -78,24 +78,26 @@
     "EvalML natively supports one-hot encoding. Here we keep 1 out of the 6 categorical columns to decrease computation time."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because the fraud labels are binary, we will use `AutoMLSearch(problem_type='binary')`. When we call `.search()`, the search for the best pipeline will begin. "
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "X = X.drop(['datetime', 'expiration_date', 'country', 'region', 'provider'], axis=1)\n",
+    "cols_to_drop = ['datetime', 'expiration_date', 'country', 'region', 'provider']\n",
+    "for col in cols_to_drop:\n",
+    "    X.pop(col)\n",
```

Comment:
Currently no `drop` method in Woodwork, so we pop each column from the DataTable :d